src/components/apps/launch/ParamGroups.js

```diff
@@ -123,28 +122,35 @@ const buildParamId = (baseId, paramIndex, type) => {
 /**
  * Form fields and info display for an app parameter group.
  */
-const ParamGroupForm = withI18N((props) => {
-    const classes = useStyles();
-
+function ParamGroupForm(props) {
     const {
         baseId,
         fieldName,
+        noOfGroups,
+        index,
         group,
         referenceGenomes,
         referenceGenomesLoading,
         startingPath,
     } = props;
+    const classes = useStyles();
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
 
     return (
-        <Accordion id={baseId} defaultExpanded>
-            <AccordionSummary
-                expandIcon={
-                    <ExpandMore id={buildDebugId(baseId, ids.BUTTONS.EXPAND)} />
-                }
+        <>
+            <Divider className={classes.paramsSectionHeader} />
+            <Typography
+                variant={isMobile ? "subtitle2" : "h6"}
+                component="span"
             >
-                <Typography variant="subtitle1">{group.label}</Typography>
-            </AccordionSummary>
-            <AccordionDetails className={classes.accordionDetails}>
+                {getMessage("section", {
+                    values: { groupNumber: index + 1, totalGroups: noOfGroups },
+                })}
+                {group.label}
+            </Typography>
+            <Divider className={classes.paramsSectionHeader} />
+            <Grid container spacing={3}>
```

Comment:
`Grid` layout allows the parameters fields to expand to fill the screen according to screen size.